strmangle.go

```diff
@@ -720,6 +723,20 @@ func ReplaceReservedWords(word string) string {
 	return word
 }
 
+// ReplaceNonAlphaNumericChars replaces non-alphanumeric characters with provided substitute.
+func ReplaceNonAlphaNumericChars(s []byte, replace rune) []byte {
```

Comment:
By default, I prefer to leave things unexported until we are sure we need it to be exported.

That way we don't have to worry about causing a breaking change by modifying it.